src/js/pensions/config/form.js

```diff
@@ -776,22 +798,167 @@ const formConfig = {
               'view:hasDependents': {
                 type: 'boolean'
               },
+              // merge with definition, provide minItems: 1, and items.required: ['relationship', 'fullName']
               dependents: {
                 type: 'array',
                 minItems: 1,
                 items: {
                   type: 'object',
-                  required: ['relationship', 'fullName'],
+                  required: ['dependentRelationship', 'fullName'],
                   properties: {
-                    relationship: dependents.items.properties.relationship,
+                    dependentRelationship: dependents.items.properties.dependentRelationship,
                     fullName: dependents.items.properties.fullName,
```

Comment:
`fullName` is no longer enforcing the standard requirements (first, last). Any ideas?